src/componentManifest.ts

```diff
@@ -103,8 +89,13 @@ export function makeDeviceManifest({
       const lang: string | undefined = file.translationLanguage;
 
       if (lang) {
-        if (manifest.i18n[lang] === undefined) manifest.i18n[lang] = {};
-        manifest.i18n[lang].resources = normalizeToPOSIX(file.relative);
+        if (locales[lang] === undefined) locales[lang] = {};
+        locales[lang].resources = normalizeToPOSIX(file.relative);
+      }
+
+      const isEntryPoint: boolean | undefined = file.isEntryPoint;
```

Comment:
Why the new variable? It looks like it's only used for the if-statement condition.
```ts
if (file.isEntryPoint) {
```